examples/options/include/traccc/options/detector.hpp

```diff
@@ -30,7 +30,8 @@ class detector : public interface {
     std::string grid_file;
     /// Use detray::detector for the geometry handling
     bool use_detray_detector = false;
-
+    // Use detray geometry id (No ACTS geometry conversion)
+    bool use_detray_simulation = false;
```

Comment:
:thinking: This is meant for the input data, no? That when we read simulation files produced using Detray, we shouldn't change the IDs associated with measurements during reading... right?

I would propose to then rather put it into `traccc::opts::input_data`. :thinking:

Reply:
What you understand is correct. It is just for not changing the IDs during the reading. 

I will change the location as suggested

Reply:
I changed its name to `use-acts-geom-source` and move it to `input_data.hpp`